crates/testsuite/features/cmd.feature

```diff
@@ -67,6 +67,7 @@ Feature: Rooch CLI integration tests
       # transaction
       Then cmd: "transaction get-transactions-by-order --cursor 0 --limit 1 --descending-order false"
       Then cmd: "transaction get-transactions-by-hash --hashes {{$.transaction[-1].data[0].execution_info.tx_hash}}"
+      Then cmd: "transaction submit --signed-tx-hex {{$.transaction[-1]}}"
```

Comment:
Also need an assert

Reply:
> Also need an assert

Resolved. Should #2473 be merged, I'll rebase this PR and merge this.